packages/x-components/src/components/__tests__/global-x-bus.spec.ts

```diff
@@ -1,36 +1,41 @@
-import { mount, Wrapper } from '@vue/test-utils';
+import { mount } from '@vue/test-utils';
 import Vue, { ComponentOptions } from 'vue';
 import { installNewXPlugin } from '../../__tests__/utils';
 import GlobalXBus from '../global-x-bus.vue';
+import { bus } from '../../plugins/x-bus';
 
-function renderGlobalXBus({ listeners = {} }: RenderGlobalXBusOptions = {}): RenderGlobalXBusAPI {
+function renderGlobalXBus({ listeners = {} }: RenderGlobalXBusOptions = {}): void {
   const [, localVue] = installNewXPlugin();
```

Comment:
The x plugin is getting deprecated, so don't use `installNewXPlugin` (not even sure if you need it for this test)